techmatters/flex-plugins
# Chi 1972 summary tab ability add contact case

Primary reviewer: @stephenhand 

## Description

- New Feature: This is the final part for this [PR](https://github.com/techmatters/flex-plugins/pull/1788) that addresses how the search results that appear should be.

### Checklist

### Related Issues
Fixes [CHI-1972](https://tech-matters.atlassian.net/browse/CHI-1972)

### Verification steps

- Create an offline contact
- Verify that we have a new "Add to New Case” drop-up button with "New Case" and "Existing Case" as options
- The "New Case” button should behave the same as the previous “Add Contact to Case” button
- The “Existing Case” button should take you to the Search tab where the header would be "Search for Cases"
- Search for a Case
- Verify that the search results only return cases


[CHI-1972]: https://tech-matters.atlassian.net/browse/CHI-1972?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
